default_module_facts.yml

```diff
@@ -1,2 +1,4 @@
 ---
 pe_version: '2016.4.1'
+puppet_environmentpath: '/etc/puppetlabs/code/environments'
+puppet_vardir: '/opt/puppetlabs/puppet/cache'
```

Comment:
Shouldn't this file go in `spec`? That's where [selinux](https://github.com/oranenj/puppet-selinux/blob/36a6b4f1ed540595d67257a972032a605c6321fa/spec/default_module_facts.yml) puts it.